infra/azure/k8s/cilium/kube-system.yaml

```diff
@@ -0,0 +1,221 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: default-deny
+  namespace: kube-system
+spec:
+  endpointSelector: {}
+  ingress:
+    - {}
+  egress:
+    - {}
+---
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: kube-dns
+  namespace: kube-system
+spec:
+  endpointSelector:
+    matchLabels:
+      k8s-app: kube-dns
+  ingress:
+    # Allow kube-dns to receive DNS requests from any endpoint in the cluster
+    # Policies for those endpoints will determine if and how they can communicate with kube-dns
+    - fromEntities:
+        - cluster
+      toPorts:
+        - ports:
+          - port: "53"
+            protocol: ANY
+    # Allow konnectivity-agent to communicate with kube-dns
+    - fromEndpoints:
+        - matchLabels:
+            app: konnectivity-agent
+            k8s:io.kubernetes.pod.namespace: kube-system
+      toPorts:
+        - ports:
+          - port: "9153"
+            protocol: ANY
```

Comment:
Does this one seem AKS specific? Or, at least not common to all K8s implementations?